src/database/UIDatabase.js

```diff
@@ -127,6 +127,8 @@ class UIDatabase {
         return results.filtered('type == $0', 'receipt');
       case 'Payment':
         return results.filtered('type == $0', 'payment');
+      case 'CashTransaction':
```

Comment:
A workaround the fact that realm doesn't allow concatenation of result objects. Seems a bit redundant to combine two existing types, but at the moment can't think of a better way of doing this.